Sources/Utility/Git.swift

```diff
@@ -66,9 +67,14 @@ public class Git {
             return (try? Git.runPopen([Git.tool, "-C", path, "tag", "-l"]))?.hasPrefix("v") ?? false
         }
 
-        public func fetch() throws {
+        public func fetch(quick: Bool = true) throws {
             do {
-                try system(Git.tool, "-C", path, "fetch", "--tags", "origin", message: nil)
+                if quick {
+                    // this probably is not a good idea
+                    try system(Git.tool, "-C", path, "fetch", "origin", "refs/tags/*:refs/tags/*", message: nil)
+                } else {
+                    try system(Git.tool, "-C", path, "fetch", "--tags", "origin", message: nil)
```

Comment:
I think so, as it's the default name that git uses when cloning, isn't it? And if we're in control of cloning, I think it should be fine. Or is there another reason, @mxcl?